MRIBase/src/Datatypes/Flags.jl

```diff
@@ -0,0 +1,96 @@
+export FLAGS, create_flag_bitmask, flag_is_set, flag_set!, flag_remove!, flag_remove_all!
+FLAGS = Dict(
```

Comment:
If we ever need more than 64 flags (or you want to not have to fiddle around with bits) I think we could also make use of BitArrays/BitVectors

Reply:
The number of flags follow the convention from ismrmrd format : https://ismrmrd.readthedocs.io/en/latest/mrd_raw_data.html#mrd-acquisitionflags

and the implementation mimics the one in the matlab code of the repository : https://github.com/ismrmrd/ismrmrd/blob/master/matlab/%2Bismrmrd/AcquisitionHeader.m